h/static/scripts/frame-rpc.js

```diff
@@ -34,81 +34,81 @@ var VERSION = '1.0.0';
 module.exports = RPC;
 
 function RPC (src, dst, origin, methods) {
```

Comment:
Probably, seeing as the comment above says `(see git log for the modifications),` and the git log is going to be pretty unhelpful when every line has changed in this commit.